lib/presentation/widgets/movie_list_presentation/grid_widgets/text_title_grid.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+
+import '../../../../core/utils/constants/dimens.dart';
+
+class TextTitleGrid extends StatelessWidget {
```

Comment:
little change here to GridTextTitle for better readability , same for the file name

Reply:
Done!